### ISSUE TITLE
[FEATURE] Adding field on registration form 

### ISSUE THREAD

**Issue by Zappie080011** [Attendize/Attendize#1114]
Is there any way I can add a "Department" field in the event registration form? Not only on the form but in the ticket design as an identifier for easy filter on excel..

**Comment by rootjack404** [comment]
You could do that pretty easily, It's just a matter of adding form on the CheckOut, creating the rows on the database table, and adding it to models, and modifying a bit of ManageEvent form to show them on the table. After all that you just need to add it in the Ticket templates for showing it in the design.

**Comment by Zappie080011** [comment]
> You could do that pretty easily, It's just a matter of adding form on the CheckOut, creating the rows on the database table, and adding it to models, and modifying a bit of ManageEvent form to show them on the table. After all that you just need to add it in the Ticket templates for showing it in the design.

Hmmmm just like that? Then I just need to know what files or line of code should i be changing or improve.. thanks for the response

**Comment by rootjack404** [comment]
> > You could do that pretty easily, It's just a matter of adding form on the CheckOut, creating the rows on the database table, and adding it to models, and modifying a bit of ManageEvent form to show them on the table. After all that you just need to add it in the Ticket templates for showing it in the design.
> 
> Hmmmm just like that? Then I just need to know what files or line of code should i be changing or improve.. thanks for the response

It will probably too long, and might be different from what I did, just do trial n error, you will figure it out

**Comment by Zappie080011** [comment]
> > > You could do that pretty easily, It's just a matter of adding form on the CheckOut, creating the rows on the database table, and adding it to models, and modifying a bit of ManageEvent form to show them on the table. After all that you just need to add it in the Ticket templates for showing it in the design.
> > 
> > 
> > Hmmmm just like that? Then I just need to know what files or line of code should i be changing or improve.. thanks for the response
> 
> It will probably too long, and might be different from what I did, just do trial n error, you will figure it out

Yeah it will be a challenge for me.. I'll just have to find the specific files i have to edit and I am now enjoying the trial and error progress..

**Comment by wboote** [comment]
autogen feature from cursor:

Here is a Department field added.  You can use git to apply the .patch files (do them in num order)

[0001-cursor-first-pass-at-implmententation.patch](https://github.com/user-attachments/files/19768902/0001-cursor-first-pass-at-implmententation.patch)

[0002-division-wasn-t-being-stored.patch](https://github.com/user-attachments/files/19768906/0002-division-wasn-t-being-stored.patch)
